omod/src/main/java/org/openmrs/module/coreapps/fragment/controller/clinicianfacing/VisitsSectionFragmentController.java

```diff
@@ -139,8 +139,11 @@ public void controller(FragmentConfiguration config,
 		}
 
 		Map<VisitDomainWrapper, String> recentVisitsWithLinks = new LinkedHashMap<VisitDomainWrapper, String>();
+
+		if (!recentVisits.isEmpty()) {
+			contextModel.put("visit", new VisitContextModel(recentVisits.get(recentVisits.size() - 1)));
+		}

```

Comment:
Not sure why this change was necessary?  Not really a problem, but I'm curious what this achieved.

Reply:
Just made the code more readable, more plainly expressive of what's actually happening.